src/util/prolog/prologResult.ts

```diff
@@ -67,12 +91,18 @@ export class PrologResult {
 		return fetchAllMatchesForAGroup(source, pattern, 1);
 	}
 
+	/**
+		* @return {Set<string>} All variable which are assigned to in the raw results
+		*/
```

Comment:
Personally, I like to align the stars below each other, but I'd say that's up to preference.

Reply:
Sometimes github changes the formatting completely for me. I don't know why.
I hope it looks proper when cloned?
I think you could call these kind of alignments an "act of violence". ^^

This is what they rather look like:
![uplaod](https://user-images.githubusercontent.com/17751957/58806450-7dd4a080-8616-11e9-88fd-8cb40cfebf97.JPG)

Reply:
It is probably the different tab-settings on the different systems. In our project, I set the tab-width to 2, whereas github uses 4 by default. You can add `?ts=2` to the end of the browser-location (like https://github.com/diproche/webinterface/pull/34/files?ts=2) to temporarily get the tabs "right".